src/core/Table/columns/index.ts

```diff
@@ -0,0 +1,5 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+export { ACTION_CELL_ID, ActionColumn } from './actionColumn';
```

Comment:
hmm..Not sure we need to export ID.. You are not using it anywhere outside.
Is there a use case that users need that?

Reply:
Removed